app/concepts/cinema_hall/cinema_hall_repository.rb

```diff
@@ -0,0 +1,5 @@
+module Repositories
+    class CinemaHallRepository  < ::Repositories::Base
```

Comment:
You could add here initializer with `CinemaHall` as an adapter. You probably won't initialize this class with another adapter than above. You can also change the class name to just `Repository` and place it within the `CinemaHall` module. 
```  
module CinemaHall
    class Repository
    end
end
``` 
This applies also to other repository classes it the project.